OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -324,6 +323,13 @@
     <Compile Include="Traits\Parachutable.cs" />
     <Compile Include="Traits\ParaDrop.cs" />
     <Compile Include="Traits\Passenger.cs" />
+    <Compile Include="Traits\Multipliers\UpgradableMultiplierTrait.cs" />
+    <Compile Include="Traits\Multipliers\DamageMultiplier.cs" />
+    <Compile Include="Traits\Multipliers\FirepowerMultiplier.cs" />
+    <Compile Include="Traits\Multipliers\InaccuracyMultiplier.cs" />
+    <Compile Include="Traits\Multipliers\ReloadTimeMultiplier.cs" />
```

Comment:
Nit: The weapon property is called ```ReloadDelay```, so it would be more consistent if this was called ```ReloadDelayModifier```.